XMonad/Layout/BorderResize.hs

```diff
@@ -112,26 +109,26 @@ handleGone wrsGone = mapM_ deleteWindow borderWins
     where
         borderWins = map bWin . concatMap snd . M.elems $ wrsGone
 
-handleAppeared :: M.Map Window Rectangle -> X (M.Map Window RectWithBorders)
-handleAppeared wrsAppeared = do
+handleAppeared :: Dimension -> M.Map Window Rectangle -> X (M.Map Window RectWithBorders)
```

Comment:
As with the earlier one, this should probably be provided as a variant with a tick appended, and the same with the following functions.

Reply:
All of these are internal functions, so it's not necessary to have variants here